docs/developers/semantic-type.md

```diff
@@ -49,7 +49,7 @@ Use `MethodSignature.returnType` to get the types of vals.
 
 ```scala mdoc
 def getType(symbol: Symbol): SemanticType =
-  symbol.info.get.signature match {
+  (symbol.info.get.signature: @unchecked) match {
```

Comment:
maybe it's clearer with a `case _` ?

Reply:
That would make the doc harder to read indeed... Does removing the `--unchecked` from `scalacOption` for the doc project silence these exceptions? If not, there is https://github.com/scala/scala/pull/8373 that could be used as well to avoid repeating this everywhere (which should be usable since the doc is built using 2.13?).

Reply:
removing -unchecked doesn't silence these exceptions